.configure

```diff
@@ -1,7 +1,7 @@
 {
   "project_name": "simplenote-android",
   "branch": "trunk",
-  "pinned_hash": "22b584c6e99946ef9fb20ce62b8f729c5bea16b7",
+  "pinned_hash": "1fa6315925665d577d1570adb56906c4ed47bc2c",
```

Comment:
This updates `gradle.properties` to include the new location of the debug keystore.

I left a comment about this [here](https://github.com/woocommerce/woocommerce-android/pull/4512#issuecomment-890732951), too.